docs/root/faq/edge.rst

```diff
@@ -0,0 +1,85 @@
+.. _faq_edge:
+
+Can I use Envoy as an edge proxy?
```

Comment:
Can we reconcile with this page? https://www.envoyproxy.io/docs/envoy/latest/configuration/best_practices/best_practices

I don't have a strong opinion on what we do here in terms of moving this to there, but I do think we should retain a FAQ no matter what and if we move this text there, link to there from here. cc @htuch who made that placeholder for additional thoughts.

Reply:
Yeah, +1 on best practices. We want parallel track for performance and security, these will inform out work on default profiles further down the track.

Reply:
Moved to Best Practices section.